ui-v2/app/adapters/intention.js

```diff
@@ -9,7 +9,12 @@ export default Adapter.extend({
   requestForQuery: function(request, { dc, filter, index, uri }) {
     return request`
       GET /v1/connect/intentions?${{ dc }}
-      X-Request-ID: ${uri}
+      X-Request-ID: ${uri}${
+      typeof filter !== 'undefined'
+        ? `
+      X-Range: ${filter}`
+        : ``
+    }
```

Comment:
Ouch just found out how `prettier` has 'prettified' our nice one-liner conditional, it looked nicer than this beforehand honest! Joking aside, I think this is fine, we could add some prettier ignores in here but I don't think its worth it.